RTN-004.tex

```diff
@@ -53,7 +57,320 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
- 
+\section{Introduction to Data Preview 0}\label{sec:intro}
+
+There are three planned Data Previews (DPs) during the period leading up to the start of Rubin Operations, termed pre-operations.
+DP0 will be based on simulated data.
+DP1 and DP2 will be based on data taken during Rubin Observatory commissioning; DP1 based on data taken with the Rubin Observatory Commissioning Camera (ComCam), and DP2 based on data taken with the LSST Camera (LSSTCam) -- the science instrument for Rubin Observatory.
+DP0 is planned for 2021 with DP1 and DP2 currently foreseen for 2022 and 2023 respectively.
+The Data Previews will enable the community to access and analyze LSST-like data in the Rubin Science Platform (RSP) on a shared-risk basis. 
+
+\subsection{The Goals of DP0}\label{ssec:intro_goals}
+
+As described in \citeds{RTN-001}, the two goals of DP0 are:
+\begin{enumerate}
+\item To serve as an early integration test of the Data Management System (DMS) and inform further development of Rubin?s LSST Science Pipelines (hereafter LSST Science Pipelines) and the Rubin Science Platform (RSP).
+\item To prepare the community to use the LSST data products and services, and especially to enable early science with commissioning and year 1 data.
+\end{enumerate}
+
+In order to achieve these goals, community participation in DP0 has two main components:
+(1) performing scientific exercises with the data products and services in the RSP, and
+(2) taking action to inform or enhance the performance of the DMS, the RSP, or the science community.
+
+Participants from the science community, "DP0 delegates", will be data rights holders\footnote{Data rights holders are US or Chilean scientists, or individuals associated with an International Contribution (in-kind) proposal in progress, as defined in \citeds{RDO-013}.} who represent a diversity of backgrounds.
+The term "delegates" reflects the fact that these individuals will be representing the science community as learners and as contributors to the achievement of the DP0 goals, as an essential step along the path towards the success of the LSST. 
+
+The nature of the two goal-driven components of DP0 participation listed above will vary according to individual delegates' interests and experience.
+
+\subsection{Two Stages: DP0.1 and DP0.2}\label{ssec:intro_stages}
+
+The simulated data set for DP0 was generated by the Dark Energy Science Collaboration (DESC) and is being used under an agreement between the DESC and Rubin.
+This data set is the second in a series of data challenges, and is thus named DC2.
+More information about DC2 can be found in the DESC DC2 paper \citep{2020arXiv201005926L} and on the DESC website\footnote{\url{https://lsstdesc.org/DC2-production/Documents/}}.
+
+The DP0.1 data set will contain the DC2 images and catalogs processed by the DESC.
```

Comment:
The problem with this section is that it allows the reader to form the impression that DP0.1 and DP0.2 are equivalent capabilities - in particular that the only difference is that in DP0.2 we have processed the data whereas in DP0.1 DESC have processed the data. In fact there are significant capability differences between the two phases, such as that in DP0.1 image access is only available in nublado via the Butler whereas in DP0.2 it is available through VO services; that in DP0.1 the portal is only available for catalogues, whereas in DP0.2 it is also available for images, etc. Suggest a clarification and/or a reference to RTN-001

Reply:
I've attempted to address this by adding the following text to the bottom of Section 1.3 "Two Stages: DP0.1 and DP0.2" in the branch tickets/preops-283.

There will also be a few capability differences between the two phases, which might affect some delegates' planned activities.
For example, in DP0.1, the notebook aspect of the RSP will offer image access via the Butler\footnote{The Butler is a middleware component for persisting and retrieving image datasets.}, whereas in DP0.2 image access will be available via Virtual Observatory (VO) services.
In DP0.1, the portal aspect of the RSP will only be available for catalogs, whereas in DP0.2 it will also be available for images.
Further details about capability differences can be found in \citeds{RTN-001}, and will also be clarified in the resources and materials for DP0 delegates (Section \ref{ssec:res_sup}).